src/makeRobotAccountant.js

```diff
@@ -7,6 +7,17 @@
 
 function makeRobotAccountant() {
   // write code here
-}
+  let callCount = 0;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      callCount++;
+
+      return callCount < 4 || callCount & 1
```

Comment:
```suggestion
      return callCount < 3 || callCount % 2 === 1
```
why do you use ` &` here????

Reply:
Read this article 
![image](https://user-images.githubusercontent.com/116519764/212039102-ec596eb1-7b6d-4769-b8d6-832f6b20ebbc.png)

Reply:
🔥